R/install.r

```diff
@@ -62,120 +62,123 @@
 #' @export
 install <-
   function(pkg = ".", reload = TRUE, quick = FALSE, local = TRUE,
-           args = getOption("devtools.install.args"), quiet = FALSE,
-           dependencies = NA, upgrade_dependencies = TRUE,
-           build_vignettes = FALSE,
-           keep_source = getOption("keep.source.pkgs"),
-           threads = getOption("Ncpus", 1),
-           force_deps = FALSE,
-           metadata = remote_metadata(as.package(pkg)),
-           out_dir = NULL,
-           skip_if_log_exists = FALSE,
-           ...) {
-
-  pkg <- as.package(pkg)
-
-  # Forcing all of the promises for the current namespace now will avoid lazy-load
-  # errors when the new package is installed overtop the old one.
-  # https://stat.ethz.ch/pipermail/r-devel/2015-December/072150.html
-  if (is_loaded(pkg)) {
-    eapply(pkgload::ns_env(pkg$package), force, all.names = TRUE)
-  }
+             args = getOption("devtools.install.args"), quiet = FALSE,
```

Comment:
This whole file is not noticeably changed much by styler. The side-by-side compare is mixed-up for reasons I can't discern.